README.md

```diff
@@ -1,33 +1,37 @@
-<div align="center">
-    <img width="720" alt="hugegraph-logo" src="https://user-images.githubusercontent.com/17706099/149281100-c296db08-2861-4174-a31f-e2a92ebeeb72.png" style="zoom:100%;" />
-</div>
+<h1 align="center">
+    <img width="720" alt="hugegraph-logo" src="https://github.com/apache/incubator-hugegraph/assets/38098239/e02ffaed-4562-486b-ba8f-e68d02bb0ea6" style="zoom:100%;" />
+</h1>
 
-<p align="center">
-    A graph database that supports more than 10 billion data, high performance and scalability
-</p>
-<hr/>
+<h3 align="center">A graph database that supports more than 10 billion data, high performance and scalability</h3>
 
-# Apache HugeGraph
+<div align="center">
 
 [![License](https://img.shields.io/badge/license-Apache%202-0E78BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
 [![HugeGraph-CI](https://github.com/apache/incubator-hugegraph/actions/workflows/ci.yml/badge.svg)](https://github.com/apache/incubator-hugegraph/actions/workflows/ci.yml)
 [![CodeQL](https://github.com/apache/incubator-hugegraph/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/apache/incubator-hugegraph/actions/workflows/codeql-analysis.yml)
 [![License checker](https://github.com/apache/incubator-hugegraph/actions/workflows/licence-checker.yml/badge.svg)](https://github.com/apache/incubator-hugegraph/actions/workflows/licence-checker.yml)
-[![Codecov](https://codecov.io/gh/apache/hugegraph/branch/master/graph/badge.svg)](https://codecov.io/gh/apache/hugegraph)
+[![Codecov](https://codecov.io/gh/apache/incubator-hugegraph/branch/master/graph/badge.svg)](https://app.codecov.io/gh/apache/incubator-hugegraph)
 [![GitHub Releases Downloads](https://img.shields.io/github/downloads/apache/hugegraph/total.svg)](https://github.com/apache/hugegraph/releases)
+[![stars](https://img.shields.io/github/stars/apache/hugegraph)](https://github.com/apache/incubator-hugegraph/stargazers)
+
+</div>
+
+## 🚀 What is Apache HugeGraph?
```

Comment:
the icon doesn't seem to be able to accurately express the meaning, we would rather remove it, the same below (until "Contributing").

Reply:
OK, I will remove all icon to keep same style.